### PR TITLE
Fixes #27417 - Remove quotes from cron template

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: molecule/default/tests/test_default.py [hunk 1]
```diff
@@ -50,6 +50,6 @@ def test_foreman_scap_client_cron(host):
     cron = file.content_string
 
     assert re.match(
-        r'1 12 \* \* 1 root "/bin/sleep \d+; /usr/bin/foreman_scap_client 1 > /dev/null"',
+        r'1 12 \* \* 1 root "/bin/sleep \d+; /usr/bin/foreman_scap_client 1 > 2>&1 | logger -t foreman_scap_client',
```
Comment (evgeni) [inline molecule/default/tests/test_default.py]:
```suggestion
        r'1 12 \* \* 1 root /bin/sleep \d+; /usr/bin/foreman_scap_client 1 > 2>&1 | logger -t foreman_scap_client',
```

File: templates/cron.j2 [hunk 2]
```diff
@@ -6,5 +6,5 @@
 
 # foreman_scap_client cron job
 {% for policy in foreman_scap_client_policies %}
-{{ policy['minute'] }} {{ policy['hour'] }} {{ policy['monthday'] }} {{ policy['month'] }} {{ policy['weekday'] }} root "/bin/sleep {{ foreman_scap_client_cron_splay | random(seed = foreman_scap_client_cron_splay_seed) if foreman_scap_client_cron_splay_seed != '' else foreman_scap_client_cron_splay | random }}; /usr/bin/foreman_scap_client {{ policy['id'] }} > /dev/null"
+{{ policy['minute'] }} {{ policy['hour'] }} {{ policy['monthday'] }} {{ policy['month'] }} {{ policy['weekday'] }} root /bin/sleep {{ foreman_scap_client_cron_splay | random(seed = foreman_scap_client_cron_splay_seed) if foreman_scap_client_cron_splay_seed != '' else foreman_scap_client_cron_splay | random }}; /usr/bin/foreman_scap_client {{ policy['id'] }} > 2>&1 | logger -t foreman_scap_client
```
Comment (evgeni) [inline templates/cron.j2]:
```suggestion
{{ policy['minute'] }} {{ policy['hour'] }} {{ policy['monthday'] }} {{ policy['month'] }} {{ policy['weekday'] }} root /bin/sleep {{ foreman_scap_client_cron_splay | random(seed = foreman_scap_client_cron_splay_seed) if foreman_scap_client_cron_splay_seed != '' else foreman_scap_client_cron_splay | random }}; /usr/bin/foreman_scap_client {{ policy['id'] }} 2>&1 | logger -t foreman_scap_client
```

File: molecule/default/tests/test_default.py [hunk 3]
```diff
@@ -50,6 +50,6 @@ def test_foreman_scap_client_cron(host):
     cron = file.content_string
 
     assert re.match(
-        r'1 12 \* \* 1 root "/bin/sleep \d+; /usr/bin/foreman_scap_client 1 > /dev/null"',
+        r'1 12 \* \* 1 root /bin/sleep \d+; /usr/bin/foreman_scap_client 1 > 2>&1 | logger -t foreman_scap_client',
```
Comment (evgeni) [inline molecule/default/tests/test_default.py]:
```suggestion
        r'1 12 \* \* 1 root /bin/sleep \d+; /usr/bin/foreman_scap_client 1 2>&1 | logger -t foreman_scap_client',
```

**Comment by xprazak2** [comment]
All green, merging. Thanks everyone!

**Comment by san7ket** [comment]
Testing looks good. Template was substituted successfully and command was success

**Pull request merged**